local_pathfinding/node_navigate.py

```diff
@@ -141,6 +154,35 @@ def get_desired_heading(self) -> float:
         # TODO: create function to compute the heading from current position to next local waypoint
         return 0.0
 
+    def LPath_callback(self):
+        """Get and publish the local path."""
+
+        waypoints = self.get_LPath()
+
+        local_path = Path(waypoints=waypoints)
```

Comment:
It might be nice to rename the `waypoints` variable to `current_waypoints` or something, so line 160 is less confusing. I might just be traumatized from trying to learn C++ tho so up to you haha